test/data/test_multi_embedding_tensor.py

```diff
@@ -79,6 +79,42 @@ def test_size():
     assert met.shape[2] == -1
 
 
+def test_fillna_col():
+    # Creat a MultiEmbeddingTensor containing all -1's
+    # In MultiEmbeddingTensor with torch.long dtype,
+    # -1's are considered as NaNs.
+    tensor_list = [
+        torch.tensor([[-1, -1, -1], [-1, -1, -1]]),
+        torch.tensor([[-1, -1], [-1, -1]]),
+    ]
+    met_with_nan = MultiEmbeddingTensor.from_tensor_list(tensor_list)
+
+    # Test fillna_col
+    for col in range(met_with_nan.num_cols):
+        met_with_nan.fillna_col(col, col)
+    assert not torch.all(met_with_nan.values == -1).any()
+    for col in range(met_with_nan.num_cols):
+        column = met_with_nan[:, col]
+        assert torch.all(column.values == col)
+
+    tensor_list = [
+        torch.tensor([[torch.nan, torch.nan, torch.nan],
+                      [torch.nan, torch.nan, torch.nan]]),
+        torch.tensor([[torch.nan, torch.nan], [torch.nan, torch.nan]]),
+    ]
```

Comment:
can we add a bit of non-nan values in `tensor_list` (at least one for each colun)?

Reply:
Sure.

Reply:
Added. Are you fine with non-nan values in each column equal the `fill_value`?

Reply:
Ideally different.

Reply:
Makes sense. Then it can test that the `fillna_col` is not filling the non_nan values. I have updated my code to reflect this.